stories/charts.js

```diff
@@ -102,34 +103,34 @@ const customizedColorMap = [
 
 storiesOf('Chart', module)
   .add('SummaryHorizontalBarChart', () => (
-    <SummaryHorizontalBarChart data={virusData} title='bar chart title' showPercentage={false} />
+    <SummaryHorizontalBarChart data={virusData} title='bar chart title' showPercentage={false} totalCount={virusDataTotal}/>
   ))
   .add('SummaryHorizontalBarChart with single color and percentage', () => (
-    <SummaryHorizontalBarChart data={virusData} title='bar chart title' color='#3283c8' />
+    <SummaryHorizontalBarChart data={virusData} title='bar chart title' color='#3283c8'  totalCount={virusDataTotal} />
   ))
   .add('SummaryHorizontalBarChart with customized colors', () => (
-    <SummaryHorizontalBarChart data={virusData} title='bar chart title' showPercentage={false} useCustomizedColorMap customizedColorMap={customizedColorMap} />
+    <SummaryHorizontalBarChart data={virusData} title='bar chart title' showPercentage={false} useCustomizedColorMap customizedColorMap={customizedColorMap}  totalCount={virusDataTotal}/>
```

Comment:
there are multiple eslint issues in this file

Reply:
dont know why my linter does not lint this file, but i ran it manually